src/Entity/IdeasWorkshop/Question.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\EntityNameTrait;
+use AppBundle\Entity\EntityPositionTrait;
+use AppBundle\Entity\EntityPublishableTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity
+ *
+ * @UniqueEntity("name")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class Question
+{
+    use EntityNameTrait;
+    use EntityPublishableTrait;
+    use EntityPositionTrait;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var Guideline
+     *
+     * @ORM\ManyToOne(targetEntity="Guideline", inversedBy="questions")
+     */
+    private $guideline;
+
+    /**
+     * @ORM\Column
+     */
+    private $placeholder;
+
+    /**
+     * @ORM\OneToOne(targetEntity="Answer", mappedBy="question")
+     */
+    private $answer;
+
+    public function __construct(string $name, bool $publishable = false)
+    {
+        $this->setName($name);
```

Comment:
Same here, you can set directly the property.

Reply:
:heavy_check_mark: